documentation/source/user_section/tutorials/spinalcord-segmentation/sct_propseg-example-t1.rst

```diff
@@ -0,0 +1,44 @@
+Hands-on Example: T1
+####################
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t1_image.png
+  :align: right
+  :figwidth: 8%
+
+  t1.nii.gz
+
+Next, we will navigate to the T1 directory and verify that it contains a single T1-weighted image. If you are still in the T2 directory from the previous section, this can be done as follows:
+
+.. code:: sh
+
+   cd ../t1
+   ls
+   # Output
+   # t1.nii.gz
+
+Once here, we can run the ``sct_propseg`` command to process the image:
+
+.. code:: sh
+
+   sct_propseg -i t1.nii.gz -c t1 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image
+   - ``-c`` : Contrast of the input image
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the segmentation slice-by-slice
+
+:Output files/folders:
+   - ``t1_seg.nii.gz`` : 3D binary mask of the segmented spinal cord
+
+Inspecting the results using QC
+*******************************
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t1_propseg_before_after.png
+  :align: right
+  :figwidth: 20%
```

Comment:
with the new RTD theme i would suggest to move the figure as a new line and center them (and make them bigger).